### DISCUSSION TITLE
Pull request issue

### DISCUSSION THREAD

**Discussion by vijaychandra1910** [recodehive/awesome-github-profiles#745] [Q&A]
I am getting that type of message please give idea how to resolve it. Give me some guidance @sanjay-kv 
![Screenshot 2024-10-11 141827](https://github.com/user-attachments/assets/361d61b9-cd7e-4617-a70c-0232ad094f52)

**Comment by Piyush-t24** [comment]
The repository owner may have chosen to opt-out of Hacktoberfest. Repository maintainers can opt out by adding the hacktoberfest topic to the repository and marking it as "not participating
